data_science/scripts/create_tunnel_to_asg.py

```diff
@@ -3,35 +3,48 @@
 """
 Connect to the first instance in an autoscaling group.
 
-Usage: create_tunnel_to_asg.py --key=<KEY> [--port=<PORT>]
+Usage: create_tunnel_to_asg.py [--key=<KEY>] [--port=<PORT>] [--type=<INSTANCE_TYPE>]
 
 Actions:
-  --key=<KEY>     Path to an SSH key with access to the instances in the ASG.
-  --port=<PORT>   Local port to use for the remote Jupyter notebook
-                  (default: 8888).
+  --key=<KEY>             Path to an SSH key with access to the instances in the ASG.
+  --port=<PORT>           Local port to use for the remote Jupyter notebook
+                          (default: 8888).
+  --type=<INSTANCE_TYPE>  AWS Instance type (valid values: p2,t2) defaults to t2
 
 """
 
 import os
 import subprocess
 import sys
 
+from os.path import expanduser
+
 import boto3
 import docopt
 
 from asg_utils import discover_data_science_asg
 
 
+def _default_ssh_key_path():
+    return '%s/.ssh/wellcomedigitalplatform' % expanduser("~")
+
+
 if __name__ == '__main__':
     args = docopt.docopt(__doc__)
 
-    key_path = os.path.abspath(args['--key'])
+    key_path = args['--key'] or _default_ssh_key_path()
+
+    print(key_path)
```

Comment:
Better: print a complete sentence that explains the significance of this path! e.g.

```python
print('Using SSH key at path %r' % path)
```